core/transaction/transaction.go

```diff
@@ -1,11 +1,11 @@
 package transaction
 
-import (
-	gogoproto "github.com/cosmos/gogoproto/proto"
-)
-
 type (
-	Msg      = gogoproto.Message
+	Msg = interface {
```

Comment:
Can we add a comment saying we use structural typing for compatibility (so it doesn't get refactored away by mistake)?